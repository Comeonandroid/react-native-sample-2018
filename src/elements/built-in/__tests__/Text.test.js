import 'react-native'
import React from 'react'
import { itBehavesLike } from '../../../specs/shared_examples/index'

import Text from '../Text'

describe('<Text />', () => {
  itBehavesLike('aThemedComponent', { subject: <Text/> })
  itBehavesLike('aComponentWithChildren', { subject: <Text /> })
})