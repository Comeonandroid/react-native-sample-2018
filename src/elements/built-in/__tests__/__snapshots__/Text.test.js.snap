// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> dark: renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#fcfcfc",
        "fontFamily": "Open Sans",
      },
      undefined,
    ]
  }
/>
`;

exports[`<Text /> light: renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#000000",
        "fontFamily": "Open Sans",
      },
      undefined,
    ]
  }
/>
`;

exports[`<Text /> renders component when passed in 1`] = `
<Styled(Text)>
  <Component />
</Styled(Text)>
`;

exports[`<Text /> renders text when passed in 1`] = `
<Styled(Text)>
  Test
</Styled(Text)>
`;
